### PR TITLE
Quick pr followup for 00_00

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 1]
```diff
@@ -46,7 +46,7 @@ I got that she was in a rush, but she really shouldn't be running in a hospital.
 <Page13>
 "――――――"
 I couldn't raise my body, so I turned my head to look up at the window.
-My eyeballs throbbed as though they were about to split into two.
+My eyeballs throbbed as though they were about to split in two.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Common/00_00.txt:49]: "into two" is a bit more awkward to say, and seems to be less prevalent

2021-12-02-002422_1994x874_scrot

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 2]
```diff
@@ -182,7 +182,7 @@ But I could see it.
 And it scared me so much, I couldn't even walk.
 <Page51>
 Speaking of which, my ears had been frozen since then, and it had gotten quieter despite the noise.
-Only the wound on my chest reverberated like a siren.
+Only the wound in my chest reverberated like a siren.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Common/00_00.txt:185]: "Wound **in**" sounds more severe, and seems more popular
2021-12-02-002428_2231x1016_scrot

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 3]
```diff
@@ -244,7 +244,7 @@ The woman crossed her arms and said confidently.
 "Well, it must be fate for us to meet like this, so why don't you keep me company for a bit? My name's Aozaki Aoko. You?"
 <Page65>
 The woman reached out to me casually as if we were friends.
-I couldn't find any reason to refuse, so I told her my name was Tohno Shiki and let her grasp suck me off the ground. // to be rechecked
+I couldn't find any reason to refuse, so I told her my name was Tohno Shiki and let her outstretched hand engulf mine.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Common/00_00.txt:247]: I think the idea here is that his hand is so much smaller than hers, not that she is yoinking him up, but could be wrong

> 　断る理由も見当たらなくて、僕は遠野志貴と名前をつげて、吸いこまれるように手のひらを握り返した。

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 4]
```diff
@@ -306,7 +306,7 @@ I traced the line and cut out half of the tree from the middle, like cutting a p
 <Page86>
 "――――――"
 Sensei was surprised.
-I got really into it, so I thrusted my knife into a horizontal line running along the base of the tree.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Common/00_00.txt:309]: Past tense thrust is still thrust

**Review by yk1512 — APPROVED** [review]

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 5]
```diff
@@ -139,8 +139,8 @@ Hey, why does the hospital have cracks everywhere?"
 <Page40>
 The doctor sighed.
 Ever since I mentioned the lines, that kind smile of his was nowhere to be seen.
-His ability to 'switch on' had broken.
-It sure seemed like the hospital was full of so many fragile things.
+I was sure that 'switch' of his had broken.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Common/00_00.txt:142]: this is one we talked about during Orange's PR. I think this line sounds kinda weird, this was my comment:

here's orange's line:
> I was sure that his 'switch' had broken.
Seemed like this hospital was full of so many fragile things.

Problem is "this hospital was full" makes it sound like he's speaking from the present about the hospital he's currently in in the past.

    きっと切り替えスイッチが故障したのだ。
    この病院は壊れやすいものばかりらしい。

Also, 切り替えスイッチ is basically missing from the first line. We use the expression "switch on [the charm etc.]" so maybe something like

    That doctor's ability to 'switch on' had broken.
    It sure seemed like the hospital was full of so many fragile things.

note full past despite second sentence

File: script/1_0_2current_en_script/Common/00_00.txt (hunk 3 above)
Comment (Avuxo) [reply script/1_0_2current_en_script/Common/00_00.txt:247]: yeah this is right, mis-tl on my part

File: script/1_0_2current_en_script/Common/00_00.txt [hunk 6]
```diff
@@ -257,7 +257,7 @@ That I had a younger sister named Akiha, who was very quiet and always followed
 <Page69>
 How I always played with Akiha and her friends in the forest-like garden of our large mansion.
 <Page70>
-How the attic was our secret hideout. How I wasn't good at <shiri|word><tori|chain> anymore. And that I was still expected to grow taller.
+How the attic was our secret hideout. How I wasn't good at <shiri|word ><tori|chain > anymore. And that I was still expected to grow taller.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Common/00_00.txt:260]: This needs spacing - `|w o r d  >` `|c h a i n>`

**Review by Avuxo — APPROVED** [review]
other than my distaste for the "switch" line, everything looks good.

**Pull request merged**